src/modules/scope.js

```diff
@@ -111,6 +111,10 @@ export default class Scope {
     /* istanbul ignore else */
     if (target) {
       target.focus();
+
+      if (document.activeElement !== target) {
+        setTimeout(() => target.focus(), 1);
```

Comment:
1ms timeout seems sketchy. Could it take longer for the target to become ready yto receive focus? Would it make sense to use `requestAnimationFrame` instead of `setTimeout`? Would it make sense to remove the first call to `target.focus()` at line 113?

Reply:
My initial draft removed the first call but that caused an automated test to fail. I agree with you, using setTimeout is not ideal. I will look into requestAnimationFrame.

Reply:
I modified to use requestAnimationFrame and it seems to work great. Thanks for the suggestion.